terraform/terraform-modules/cognito-auth/lambda/requirements.txt

```diff
@@ -0,0 +1,4 @@
+requests==2.29.0
```

Comment:
We can use `~=2.29.0` in order to get bug-fix updates to `requests`

Reply:
Agreed. I updated it to ~=2.29.0